snap/snapcraft.yaml

```diff
@@ -71,13 +113,6 @@ parts:
                        "-Dlibjcat:introspection=false",
                        "-Dlibjcat:tests=false"]
     source: .
-    source-type: git
-    override-build: |
-      snapcraftctl build
-      echo $(git describe HEAD --always) > $SNAPCRAFT_STAGE/version
```

Comment:
As the snap builds on every single release and goes to edge channel we had wanted this.

When you get to tagged release then it includes the tags. When you don't you at least know what commit you build from.

Reply:
never mind, I see you moved it.

Reply:
I feel this part should not be merged. It was to build the local files rather than from git. I can make it a separate commit not to be merged.

Reply:
Got it; don't worry.  This is a draft, we won't merge anything until it's not a draft.  But please pull in anything you want merged sooner into separate PR's so this draft PR stays lean.